project/Dependencies.scala

```diff
@@ -7,7 +7,9 @@ object Dependencies {
 
   val Scala213 = "2.13.10" // update even in link-validator.conf
   val Scala212 = "2.12.17"
-  val ScalaVersions = Seq(Scala213, Scala212)
+  val Scala3 = "3.1.3"
+  val Scala2Versions = Seq(Scala213, Scala212)
```

Comment:
> Re scala3 in PRs, I got the understanding that we only want scala3 (same for 2.12) to be used during cros publishing. At least this is how it is right now for the scla3 builds I was working on.

We should at minimum compile with all Scala versions. We typically do that with `+Test/compile`. That should be changed here
https://github.com/akka/alpakka/blob/5228417f38211cb9bfc62bed4f1f7bb2b4aed0cb/.github/workflows/check-build-test.yml#L45

It's probably obsolete to use 2.13 there, since 2.13 is the default version.

Also, currently I think we run tests with the default 2.13, which is good, but it means that we currently don't build with 2.12 at all (unless I missed something, didn't look so close). Anyway, that is fixed by compiling with `+Test/compile`.

Reply:
Makes sense, will review.